### DISCUSSION TITLE
How the "Backlog" number is calculated ?

### DISCUSSION THREAD

**Discussion by a-lebleu** [InfotelGLPI/mydashboard#122] [Q&A]
Hello !

I would like to know what does the Backlog mean and how it is calculated. I didn't find the answer and couldn't figure how to calculate it.

![image](https://user-images.githubusercontent.com/77343435/146548068-9c9a5a91-f4b5-4e22-9a99-b2f11598a708.png)

Regards,

**Comment by tsmr** [comment]
It's not a mydahsboard grah - sorry go to https://plugins.glpi-project.org/#/plugin/dashboard project
